app/deliver_grant_funding/templates/deliver_grant_funding/grant_settings.html

```diff
@@ -5,28 +5,32 @@
   Settings - {{ grant.name }} - MHCLG Funding Service
 {% endblock pageTitle %}
 
-{% set active_sub_navigation_tab = "grant_settings" %}
+{% set active_item_identifier = "settings" %}
 
 {% block content %}
-  <h1 class="govuk-heading-l">
-    <span class="govuk-caption-l">{{ grant.name }}</span>
-    Settings
-  </h1>
-  {{
-    govukSummaryList({
-          "rows": [{
-              "key": {"text": "Name"},
-              "value": {"text": grant.name},
-              "actions": {
-                  "items": [
-                      {
-                        "href": url_for("deliver_grant_funding.grant_change_name", grant_id=grant.id),
-                        "text": "Change",
-                        "visuallyHiddenText": "name"
-                      }
-                  ]
-              }
-          }]
-    })
-  }}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+      <h1 class="govuk-heading-l">
+        <span class="govuk-caption-l">{{ grant.name }}</span>
+        Settings
```

Comment:
🤏 "Grant details" feels more appropriate now to line up with the nav

Reply:
yes ty